### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by searlmc1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**Note: As of December 2023, this project is deprecated and will be moved to 'archive' status in the near future**
```
Comment (kzhuravl) [inline README.md:1]: this project is deprecated and will be archived in the near future?

Comment (searlmc1) [reply README.md]: Updated. See https://github.com/ROCm/clang-ocl/pull/49/commits/a1bbfca40ecf6f1c2b3a92f0b47a38ca1c640c18 ; I'm doing this via web interface; I'm a little clunky at it

**Review by kzhuravl — APPROVED** [review]

**Review by kzhuravl — APPROVED** [review]

**Comment by dgaliffiAMD** [comment]
Looks good 👍

**Review by yxsamliu — COMMENTED** [review]
LGTM.

**Pull request merged**